src/coreclr/jit/lower.cpp

```diff
@@ -7801,43 +7801,45 @@ void Lowering::TryRetypingFloatingPointStoreToIntegerStore(GenTree* store)
 //
 // Arguments:
 //    node - GT_AND node of integral type
+//    nextNode - out parameter that represents the 'gtNext' of the given node if the transformation was successful
 //
 // Return Value:
-//    Returns the replacement node if one is created else nullptr indicating no replacement
-GenTree* Lowering::TryLowerAndNegativeOne(GenTreeOp* node)
+//    Returns the true if the transformation was successful; false if it was not.
+bool Lowering::TryLowerAndNegativeOne(GenTreeOp* node, GenTree** nextNode)
 {
     assert(node->OperIs(GT_AND));
```

Comment:
nit: Also `assert(nextNode != nullptr)` given that you're not checking for `null` below (we have a lot of methods that allow `null` for such scenarios).